korangar/src/loaders/action/mod.rs

```diff
@@ -49,6 +49,13 @@ impl AnimationState {
         self.factor = Some(movement_speed as f32 * 100.0 / 150.0);
     }
 
+    pub fn death(&mut self, client_tick: ClientTick) {
```

Comment:
Something like `dead` is more in like with the other functions on this struct

Reply:
i will change